ui/src/components/Read/index.vue

```diff
@@ -184,8 +184,10 @@ watch(() => data.code, () => {
     if (index >= 0) {
         data.codeStorage[index].params.code = data.code
     } else {
+
         route.params.code = data.code
-        data.codeStorage.push(JSON.parse(JSON.stringify(route)))
+        const clonedObject = Object.assign({}, route);
```

Comment:
After researching, I found that this code only creates a shallow copy, not a deep copy, which could lead to potential issues. Can you take a look, @Fine0830?

Reply:
https://github.com/WebReflection/flatted might help with this issue.

Reply:
The `Object. assign` can only deeply copy the first layer of json.

Reply:
sure @hanahmily and @Fine0830, will look into this and update the PR accordingly.

Reply:
Hi @Fine0830 @hanahmily, tried using structuredClone(). Am encountering some issue similar to this [here](https://stackoverflow.com/questions/72632173/unable-to-use-structuredclone-on-value-of-ref-variable). Will try to manually loop and clone each child recursively.

Reply:
Could you use `toRow` before using `structuredClone`?

Reply:
Did you try using "toRaw" as suggested in the link you shared? This solution may help resolve the problem you encountered.

Reply:
Probably, it is enough to use `toRaw` here. Transforming a responsive object generated with reactive into a non responsive regular object, then assign a value to the new variable. All operations on this object will not cause page updates.

Reply:
@Fine0830 @hanahmily, Using toRaw is still causing the issue to persist. Will see what else can be done. Will revert to a recursive solution if nothing works out.

Reply:
How do you use `toRaw`, Only toRaw(route)? If so, it is not enough. All data in the `route` need to be converted.

Reply:
Yeah, tried using toRaw(route), toRaw(route.data) and toRaw(route.value). Nothing seemed to work for me

Reply:
You could try JSON.parse (JSON. stringify()). If it doesn't work, writing a custom function might be the only solution.